### PR TITLE
1338-jsonSchemaUpdate

### LINKED ISSUE #1338: Make JSON schema run with newer AJV versions

### PR DESCRIPTION
**Pull request by TobiasNx**
This should fix #1338

### CODE REVIEW

**Comment by dr0i** [comment]
Why'd you have the 'validation.txt' commited? It's kind of an error log, isn't it? We normally don't commit error logs.

**Comment by TobiasNx** [comment]
> Why'd you have the 'validation.txt' commited? It's kind of an error log, isn't it? We normally don't commit error logs.

Have reverted the related commit: [005adb7](https://github.com/hbz/lobid-resources/pull/1344/commits/005adb7528264cae1fd0d4faf9282997d2feeaee)

File: src/test/resources/validateJsonTestFiles.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ DIRECTORY_OF_JSON_TO_VALIDATE="alma/"
 
 for version in "draft"; do
 	echo "Testing version: $version"
-	ajv test -s schemas/resource.json -r "schemas/*.json" -d "${DIRECTORY_OF_JSON_TO_VALIDATE}/*.json" --valid 2>&1
+	ajv test -s "schemas/resource.json" -r "schemas/*.json" -d "${DIRECTORY_OF_JSON_TO_VALIDATE}/*.json" -c ajv-formats --all-errors --valid 2>&1
```
Comment (acka47) [inline src/test/resources/validateJsonTestFiles.sh]:
```suggestion
	ajv test -s "schemas/resource.json" -r "schemas/!(resource).json" -d "${DIRECTORY_OF_JSON_TO_VALIDATE}/*.json" -c ajv-formats --all-errors --valid 2>&1
```

**Review by acka47 — APPROVED** [review]
I improved the validation script a bit. Otherwise +1.

**Pull request merged**